app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+    # Validation for unique and valid email addres
+    VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i.freeze
```

Comment:
As we already use `Devise` (as a dependency from `ActiveAdmin`) we can use theirs email regexp `Devise.email_regexp`

Reply:
Changed it